spec/web_helpers.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+def sign_up(email = "test@gmail.com")
```

Comment:
perhaps change to test@example.com so that we don't send any emails to a real gmail address